pino.js

```diff
@@ -354,6 +359,9 @@ function asErrValue (err) {
 
 function genLog (z) {
   return function LOG (a, b, c, d, e, f, g, h, i, j, k) {
+    if (this.filter && !this.filter(a, b, c, d, e, f, g, h, i, j, k)) {
```

Comment:
I chose `true` to mean "yes, this log message is of relevance" as this is the way `Array.prototype.filter` works. Basically translating this in my head to `retain?` or `downstream?`.

But I don't care much about `true` or `false` for discarding. Let me know what you prefer and I'll change it :).